.circleci/continue_config.yml

```diff
@@ -207,10 +207,25 @@ jobs:
           name: Run tests
           command: make engine-docker-test
           no_output_timeout: 15m
+  
+  trigger_enterprise_unit_tests:
+      docker: 
+        - image: cimg/base:2021.11
+      resource_class: small
+      steps:
+        - run:
+            name: Trigger enterprise unit tests
+            command: |
+              curl --request POST \
+                --url https://circleci.com/api/v2/project/gh/TobikoData/sqlmesh-enterprise/pipeline \
+                --header "Circle-Token: $ENTERPRISE_CIRCLE_API_TOKEN" \
+                --header "content-type: application/json" \
+                --data '{"branch":"trigger_workflow_on_post","parameters":{"run_main_pr":false, "run_core_commit":true, "sqlmesh_version":"'"$CIRCLE_BRANCH"'"}}'
```

Comment:
Once the complementary changes in enterprise are merged this changes to `"branch":"main"`--I have it set to the feature branch for testing